src/VirtualClient/VirtualClient.Actions/HPLinpack/HPLinpackExecutor.cs

```diff
@@ -42,38 +45,17 @@ public HPLinpackExecutor(IServiceCollection dependencies, IDictionary<string, IC
             this.packageManager = this.systemManagement.PackageManager;
             this.fileSystem = this.systemManagement.FileSystem;
             this.stateManager = this.systemManagement.StateManager;
+            this.cpuInfo = this.systemManagement.GetCpuInfoAsync(CancellationToken.None).GetAwaiter().GetResult();
         }
 
         /// <summary>
-        /// The user who has the ssh identity registered for.
+        /// Parameter defines whether to use perf libraries or not.
         /// </summary>
-        public string Username
+        public bool UsePerformanceLibraries
         {
             get
             {
-                string username = this.Parameters.GetValue<string>(nameof(HPLinpackExecutor.Username), string.Empty);
-                if (string.IsNullOrWhiteSpace(username))
-                {
-                    username = Environment.UserName;
-                }
-
-                return username;
-            }
-        }
-
-        /// <summary>
-        /// True if Hyperthreading is on
-        /// </summary>
-        public bool HyperThreadingOn
-        {
-            get
-            {
-                return this.Parameters.GetValue<bool>(nameof(HPLinpackExecutor.HyperThreadingOn), true);
-            }
-
-            set
-            {
-                this.Parameters[nameof(HPLinpackExecutor.HyperThreadingOn)] = value;
+                return this.Parameters.GetValue<bool>(nameof(this.UsePerformanceLibraries), true);
```

Comment:
I think we discussed to make it false before we support it on Intel, AMD and ARM altogether